doc/source/how-to/contributing.rst

```diff
@@ -0,0 +1,250 @@
+   and helpful for PyAnsys users. When in doubt, use ``import this``.
+   For Ansys code quality standards, see :ref:`Coding Style`.
+
+#. Document your contributions. Include a docstring for any added function,
+   method, or class, following :ref:`Numpydoc Docstrings`, as specified by
+   PyAnsys :ref:`Documentation Style`. Always provide at least one simple use
+   case for a new feature.
+
+#. Test your contribution. Because Python is an interpreted language, if
+   it's not tested, it's probably broken. At the minimum, include a unit
+   test for each new feature within the ``tests`` directory. Ensure that
+   each new method, class, or function has a reasonable (>80%) coverage.
+   See :ref:`Testing` for automating testing.
+
+#. Do not include any datasets for which a license is not available
+   or commercial use is prohibited.
+
+#. Review our `Code of Conduct
+   <https://github.com/pyansys/.github/blob/main/CODE_OF_CONDUCT.md>`_.
+
+
+All ``PyAnsys`` projects are hosted in `GitHub <https://www.github.com/>`_ in
+the form of :ref:`Git` repositories. GitHub is a platform not only provides
+storage for projects but also additional features like code reviews or issue
+boards.
+
+.. raw:: html
+
+     <br>
+     <div align="center">
+       <img width="50%;" src="https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png">
+     </div>
+
+
+Creating a GitHub account
+-------------------------
+To use GitHub, start by creating a new account in the platform. Follow the
+`GitHub Join Process <https://github.com/join>`_. If you require access to the
+`PyAnsys Organization <https://github.com/pyansys>`_, refer to `Alexander
+Kaszinsky <mailto:alexander.kaszinsky@ansys.com>`_.
+
+GitHub Repository Sections
+--------------------------
+
+Once you have a GitHub account and access granted to the `PyAnsys` organization,
+you will be able to interact with the different repositories. All the
+repositories contain the following sections:
+
+.. figure:: images/github_sections.png
+   :alt: GitHub repository sections
+   :align: center
+
+* ``Code``: a tree view of the project's structure.
+* ``Issues``: the issue and feature request board.
+* ``Pull requests``: a board collecting new changes to be merged.
+* ``Discussions``: dialogs about development practices to be used by project maintainers.
+* ``Actions``: the CI/CD panel showing the different available workflows.
+* ``Projects``: a panel for organizing and planning development in the repository.
+* ``Wiki``: devoted to show quick information of the project.
+* ``Security``: configurations related to security issues, vulneravilities and alerts.
+* ``Insights``: generic information about the repository and its contributors.
+* ``Settings``: allows to configure access and integration with third party tools.
+
+.. note:: 
+
+   Depending on your access level, you may not be able to see some of the
+   previous sections.
+
```

Comment:
```suggestion
```

Reply:
I've deleted because I think this information needs to be stated in the introductory sentence as per my edits.